packages/did-provider-ebsi/package.json

```diff
@@ -12,12 +12,16 @@
   "dependencies": {
     "@ethersproject/random": "^5.7.0",
     "@sphereon/ssi-sdk-ext.did-resolver-ebsi": "workspace:^",
+    "@sphereon/ssi-sdk-ext.key-utils": "workspace:^",
     "@transmute/did-key-bls12381": "0.3.0-unstable.10",
     "@veramo/core": "4.2.0",
     "@veramo/did-manager": "4.2.0",
     "@veramo/did-provider-key": "4.2.0",
+    "cross-fetch": "^4.0.0",
     "debug": "^4.3.4",
     "did-resolver": "^4.1.0",
+    "ethers": "^6.11.1",
+    "jose": "^4.14.6",
```

Comment:
Please merge other branch, because jose is still (or again?) listed as dependency and that for sure isn't compatible with RN

Reply:
Removed `jose` dependency